### PR TITLE
9608 - Author photos look a little pixelated

### PR DESCRIPTION
**Pull request by fessehaye**
# Description

image

Increased the author photo image (more then doubled) fill size so the profile pics would be less pixelated.

Best way to check is grabbing a high def stock photo from unsplash and applying it to an author photo on a pni article page

Related PRs/issues: #9608

### CODE REVIEW

File: network-api/networkapi/templates/pages/buyersguide/article_page.html [hunk 1]
```diff
@@ -72,7 +72,7 @@ <h1 class="tw-h1-heading">{{ page.title }}</h1>
             <div class="col-lg-8 tw-flex tw-py-4">
               {% if author_profile.image %}
                 <div class="tw-mr-4 tw-flex tw-items-start tw-flex-shrink-0">
-                  <img class="tw-rounded-full tw-border-2 tw-border-white" src="{% image_url author_profile.image "fill-75x75" %}" alt="{{ author_profile.name }}" title="{{ author_profile.name }}" />
+                  <img class="tw-rounded-full tw-border-2 tw-border-white tw-w-[75px] tw-h-[75px]" src="{% image_url author_profile.image "fill-225x225" %}" alt="{{ author_profile.name }}" title="{{ author_profile.name }}" />
```
Comment (mmmavis) [inline network-api/networkapi/templates/pages/buyersguide/article_page.html:75]: Should this be twice of 75 so `fill-150x150` instead?

**Review by mmmavis — CHANGES_REQUESTED** [review]
Image looks good on page! Just had a question regarding the dimension set in code.

File: network-api/networkapi/templates/pages/buyersguide/article_page.html (hunk 1 above)
Comment (fessehaye) [reply network-api/networkapi/templates/pages/buyersguide/article_page.html]: Let me know if the image still looks good after the change!

**Review by mmmavis — CHANGES_REQUESTED** [review]
@fessehaye Could you please update the code to use srcset instead? Thank you!

**Comment by fessehaye** [comment]
@mmmavis @tbrlpld ready for review now

**Review by mmmavis — APPROVED** [review]
✨ Looking great!

**Pull request merged**